ballot/models.py

```diff
@@ -52,6 +52,10 @@ def __str__(self):
 class BallotItemResponse(models.Model):
     """
     YES/NO to a referendum, or a candidate.
+
+    This is a *conceptually* abstract class, but exists
+    as a Django model so that finance can point to propositions
+    and candidates equally.
     """
```

Comment:
I'm sorry, I still don't understand what a "response" is. What does the comment "YES/NO to a referendum" mean? It sounds like how someone marks their ballot YES or NO.  I don't think we are tracking that information.  What do you mean "*conceptually* abstract class"?  In my experience an abstract class would have the minimum required fields and the actual class would subclass it adding the needed fields.  This class has fields specific to referendums only.  Where is it subclassed for a candidate?
Again why is this not a "description" rather than a "response"?
Sorry to be such a pill about this, but I worry about folks in the future trying to understand this.